web2/src/pages/settings/PlexSettingsPage.tsx

```diff
@@ -219,22 +259,221 @@ export default function PlexSettingsPage() {
+              </Select>
+              <FormHelperText>
+                Note: Only applies when downmixing to stereo.
+              </FormHelperText>
+            </FormControl>
+          </Grid>
+        </Grid>
+      </>
+    );
+  };
+
+  const renderSubtitleSettings = () => {
+    if (streamSettingsPending) {
+      return (
+        <Paper sx={{ display: 'flex' }}>
+          <Box flex={1}>
+            <Skeleton width="70%">
+              <TextField fullWidth />
+            </Skeleton>
+          </Box>
+          <Box flex={1}>
+            <Skeleton width="70%">
+              <TextField fullWidth />
+            </Skeleton>
+          </Box>
+        </Paper>
+      );
+    }
+
+    if (!streamSettings) {
+      return <div>Error</div>;
+    }
+
+    return (
+      <>
+        <Typography component="h6" sx={{ my: 2 }}>
+          Subtitle Options
+        </Typography>
+        <Grid flex="1 0 50%" container spacing={3}>
+          <Grid item xs={12} sm={6}>
+            <FormControl fullWidth>
+              <FormControlLabel
+                control={<Checkbox onChange={onSubtitleChange} />}
+                label="Enable Subtitles (Requires Transcoding)"
+              />
+              {showSubtitles && (
+                <TextField
+                  id="component-outlined"
+                  label="Subtitle Size"
+                  defaultValue={streamSettings?.subtitleSize}
+                />
+              )}
+            </FormControl>
+          </Grid>
+        </Grid>
+      </>
+    );
+  };
+
+  const renderMiscSettings = () => {
+    if (streamSettingsPending) {
+      return (
+        <Paper sx={{ display: 'flex' }}>
+          <Box flex={1}>
+            <Skeleton width="70%">
+              <TextField fullWidth />
+            </Skeleton>
+          </Box>
+          <Box flex={1}>
+            <Skeleton width="70%">
+              <TextField fullWidth />
+            </Skeleton>
+          </Box>
+        </Paper>
+      );
+    }
+
+    if (!streamSettings) {
+      return <div>Error</div>;
+    }
+
+    return (
+      <>
+        <Grid flex="1 0 50%" container spacing={3}>
+          <Grid item xs={12} sm={6}>
+            <FormControl fullWidth sx={{ my: 1 }}>
+              <TextField
+                id="component-outlined"
+                label="Max Direct Stream Bitrate (Kbps)"
+                defaultValue={streamSettings?.directStreamBitrate}
+              />
+            </FormControl>
+            <FormControl fullWidth sx={{ my: 1 }}>
+              <TextField
+                id="component-outlined"
+                label="Max Transcode Bitrate (Kbps)"
+                defaultValue={streamSettings?.transcodeBitrate}
+              />
+            </FormControl>
+          </Grid>
+          <Grid item xs={12} sm={6}>
+            <FormControl fullWidth sx={{ my: 1 }}>
+              <TextField
+                id="component-outlined"
+                label="Direct Stream Media Buffer Size"
```

Comment:
i dont think the original provides units here, so we should figure out what the units are and add them here. it's probably bytes